### PR TITLE
REMB and TWCC

### PR DESCRIPTION
**Pull request by zhuker**
intro into bandwidth estimation and simple REMB diagram

### CODE REVIEW

**Comment by Sean-Der** [comment]
Holy shit this is amazing, great to see you again @zhuker 

What do you think of having a dedicated ‘congestion control’ chapter? We can discuss the basics of watching RTT/Loss and estimating. Then we can get into how SCTP and RTP actually work.

The issue I see is the overlap in the chapters. But I am cool with merging this w/e. This is amazing info :)

**Comment by zhuker** [comment]
glad to be finally back
hm... separate chapter 🤔 
not sure if i have enough content for a whole chapter :)
you think things like "Kalman filter for congestion control" belong in the book?

> The issue I see is the overlap in the chapters.
what chapters are overlapping?

i'd merge as is and split into separate chapter when/if there's more content

File: content/docs/05-media-communication.md [hunk 1]
```diff
@@ -181,9 +181,88 @@ Also known as FEC. Another method of dealing with packet loss. FEC is when you s
...
+#### REMB
+
+A widely supported albeit never fully standardized and now considered deprecated method is called [REMB](https://tools.ietf.org/html/draft-alvestrand-rmcat-remb-03) (Receiver Estimated Maximum Bitrate).
+REMB is a special RTCP packet receiver sends to sender notifying the sender of available bandwidth. There's no(?) standard on the method used to estimate bandwidth associated with REMB, so the actual values are implementation dependent. Good starting point for research into details or REMB is [Chrome's source code](https://source.chromium.org/chromium/chromium/src/+/master:third_party/webrtc/modules/rtp_rtcp/source/rtcp_packet/remb.cc)
+
+The only useful payload in the packet is the bitrate measured in bits per second.
+Notable abiguity and source of confusion is that REMB is defined as _total_ bitrate while it is common to see webrtc libraries use it to constrain video encoding bitrate only.
+
+{{< figure src="/images/05-remb.png">}}
+![REMB](../images/05-remb.png)
```
Comment (zhuker) [inline content/docs/05-media-communication.md:205]: @Sean-Der is this cool? i wanted to have a nice image in github .md preview vs `{{ figure ...}}`

**Comment by zhuker** [comment]
@Sean-Der ping

**Comment by Sean-Der** [comment]
@zhuker sorry I have been slow, this is great! Please don't be blocked on me to merge. We can always go back and revise things.

I just don't want to repeat the same concepts in the Data and Media chapters. SCTP is rate limited so we have overlap. Maybe we can put the chapter before media+data?

* What is congestion?
* What happens when you have congestion? (Dropped packets, increased RTT, Jitter)
* Why don't all networks act the same
* Slow start
* Probation

**Comment by zhuker** [comment]
Cool

**Pull request merged**